### DISCUSSION TITLE
Volume overall size on disk limited to 30 GB

### DISCUSSION THREAD

**Discussion by facorazza** [seaweedfs/seaweedfs#5984] [Q&A]
Hello,
I have deployed a SW cluster on 3 k3s nodes using Helm with 1 master, 3 filers and 3 volumes. Each node has a 50 GB disk assigned where the volume storage is located. The master has a volumeSizeLimitMB of 1000 and the volumes have a max of 0. From the web ui I can see a total of 150 volumes available: 50 volumes per node. That should result to 50GB per node. However, if I try to fill them I can only use 30GB on each one of them. That is 30GB out of the 50GB available on disk. All volumes are being used though.
Can anybody help me figure out why I'm unable to completely utilize the disks?
Thanks!

**Comment by mtillberg** [comment]
This is discussed in the link below, but it's based on a 4 byte offset into 8 byte aligned data.

https://github.com/seaweedfs/seaweedfs?tab=readme-ov-file#object-store-architecture

weed has a option for compiling with a 5 byte offset, which expands max volume size to ~8TB.  It's available in docker with large_disk tags, currently chrislusf/seaweedfs:3.85_large_disk.

`$ docker run --rm chrislusf/seaweedfs:3.85_large_disk master`
`I0305 22:00:37.250481 master.go:163 Start Seaweed Master 8000GB 3.85 7d7e06681 at 172.17.0.2:9333`
